src/edu/ucsb/nceas/metacat/restservice/multipart/StreamingMultipartRequestResolver.java

```diff
@@ -191,6 +199,15 @@ public MultipartRequest resolveMultipart(HttpServletRequest request) throws IOEx
                 }
             }
         }
+        if (end > start && pid != null) {
+            String predicate = null;
+            if (sysmetaFirst) {
+                predicate = "with";
+            } else {
+                predicate = "without";
+            }
+            log.info(edu.ucsb.nceas.metacat.common.Settings.PERFORMANCELOG + pid + edu.ucsb.nceas.metacat.common.Settings.PERFORMANCELOG_CREATE_UPDATE_METHOD + " Write the object file from the http multipart to the disk " + predicate + " calculating the checksum" + edu.ucsb.nceas.metacat.common.Settings.PERFORMANCELOG_DURASION + (end-start)/1000);
```

Comment:
Limit long lines.

Reply:
See the commit 534ea24ba1197a4217751001f8b4eed26e2d4ccd.